modules/gdscript/gdscript.cpp

```diff
@@ -2698,6 +2698,10 @@ Ref<Resource> ResourceFormatLoaderGDScript::load(const String &p_path, const Str
 	Error err;
 	Ref<GDScript> scr = GDScriptCache::get_full_script(p_path, err, "", p_cache_mode == CACHE_MODE_IGNORE);
 
+	if (err) {
+		ERR_PRINT(vformat(R"(Failed to load script at path "%s". Error code %d.)", p_path, err));
```

Comment:
Need to check whether `ERR_PRINT_ED` also writes an error to the console, or even whether, when autoloads get loaded, these errors would be visible.

Reply:
Here it is possible to distinguish what period the error is, see https://github.com/godotengine/godot/pull/76954#issue-1705988639:


```suggestion
		if (scr.is_valid()){  
			// Errors during compilation.
		}else{ 
			// Errors during loading.
		}
```

Reply:
Great suggestion, @Rindbee